src/components/NetworkDropdown.tsx

```diff
@@ -32,7 +32,18 @@ const Item = ({
 
 const NetworkDropdown = () => {
   const [networkConfig, setNetworkConfig] = useAtom(networkConfigAtom);
-
+  const specifiedNetworkName = new URLSearchParams(window.location.search).get('network');
+  if (specifiedNetworkName !== null) {
+    if (specifiedNetworkName in networkConfigs){
```

Comment:
If you're using vscode you can install eslint and prettier plugins to auto-lint and format. Otherwise you'll have to run `yarn format`

Reply:
I guess our CI checks lint but not format, I'll have to add that